SourceEditorExtension/SIG.swift

```diff
@@ -46,7 +46,12 @@ func generate(selection: [String], indentation: String, leadingIndent: String) t
             let variableType = scanner.scanUpTo("\n") else {
                 throw SIGError.parseError
         }
-        variables.append((variableName, variableType))
+        
+        // In case multiple variables defined in a line.
+        let variableNames = variableName.components(separatedBy: ",")
+        for vname in variableNames {
```

Comment:
Variable names should be clearer. I.e. change the variable names around so the inner loop variable can be named `variableName`.

Reply:
You could also move this to the scanner part and use `scanUpTo(characterSet:)` to directly scan the variables into an array. That way there's no need to separate and trim them afterwards.